### DISCUSSION TITLE
[Feature] Add media albums along single photos

### DISCUSSION THREAD

**Discussion by mprajescu** [mediacms-io/mediacms#606] [Feature Requests]
At the moment the photo media is treated as a single item and is displayed the same way a video is showing. 
Can it be grouped into a playlist to act as an album?

**Comment by mgogoulos** [comment]
Valid point, there are good suggestions on how photos will be treated more properly (eg albums/folders, slideshow/playlist etc)

**Comment by mgogoulos** [comment]
I realize now that we need some wire frames, or at least UI/UX suggestions before we can handle this

**Comment by TheNomad11** [comment]
Any news on this? Could then be used as flickr replacement...
